dynaml/parser_test.go

```diff
@@ -37,7 +37,31 @@ var _ = Describe("parsing", func() {
 
 	Describe("merge", func() {
 		It("parses as a merge node with the given path", func() {
-			parsesAs("merge", MergeExpr{[]string{"foo", "bar"}}, "foo", "bar")
+			parsesAs("merge alice.bob", MergeExpr{[]string{"alice", "bob"},true,false, true, ""}, "foo", "bar")
```

Comment:
As we mentioned above, we understand the standard practice in the `dynaml` package is to initialize these structs inline without named fields. Despite that, we'd like you to modify this case and those added below into multi-line initializations with named fields for the sake of clarity in the specific case that there are multiple sequential boolean values provided.